kcidb_io/schema/v4.py

```diff
@@ -29,582 +29,606 @@
-            "format": "email",
-            "description":
-                "The value of the Message-ID header of the e-mail message "
-                "introducing the checked-out source code, if any. E.g. a "
-                "message with the applied patchset, or a release "
-                "announcement sent to a maillist.",
-        },
-        "comment": {
-            "type": "string",
-            "description":
-                "A human-readable comment regarding the checkout. "
-                "E.g. the checked out release version, or the subject of the "
-                "message with the applied patchset."
-        },
-        "start_time": {
-            "type": "string",
-            "format": "date-time",
-            "description":
-                "The time the checkout was started.",
-            "examples": [
-                "2020-08-14T23:08:06.967000+00:00",
-            ],
-        },
-        "contacts": {
-            "type": "array",
-            "description":
-                "List of e-mail addresses of contacts concerned with the "
-                "checked out source code, such as authors, reviewers, and "
-                "mail lists",
-            "items": {
-                "type": "string",
-                "description":
-                    "An e-mail address of a contact concerned with this "
-                    "code, e.g. an author, a reviewer, or a mail list, "
-                    "as in https://tools.ietf.org/html/rfc5322#section-3.4"
+
+    "$defs": {
+        # JSON schema for a named remote resource
+        "resource": {
+            "title": "resource",
+            "type": "object",
+            "properties": {
+                "name": {
+                    "type": "string",
+                    "description":
+                        "Resource name. Must be usable as a local file name "
+                        "for the downloaded resource. Cannot be empty. "
+                        "Should not include directories.",
+                    "pattern": "^[^/]+$",
+                },
+                "url": {
+                    "type": "string",
+                    "format": "uri",
+                    "description":
+                        "Resource URL. Must point to the resource "
+                        "file directly, so it could be downloaded "
+                        "automatically.",
+                },
             },
-        },
-        "log_url": {
-            "type": "string",
-            "format": "uri",
-            "description":
-                "The URL of a plain-text log file of the checkout attempt. "
-                "E.g. 'git am' output.",
-        },
-        "log_excerpt": {
-            "type": "string",
-            "maxLength": 16384,
-            "description":
-                "A part of the log file of the checkout attempt most "
-                "relevant to its outcome.",
+            "description": "A named remote resource",
```

Comment:
Could you keep the `"description"` property where it was, right below the `"title"`, for each object?